pkg/aro/aroign/etchosts.go

```diff
@@ -0,0 +1,232 @@
+package aroign
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	ign3types "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/openshift/installer/pkg/asset/ignition"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/pkg/errors"
+	"github.com/vincent-petithory/dataurl"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	configFileName = "aro.conf"
```

Comment:
nit, feel free to dismiss - should we call this `aro.hosts`, just to make it clear to anyone stumbling upon this file without the context what it is for?